applications/StructuralMechanicsApplication/tests/structural_mechanics_test_factory.py

```diff
@@ -38,6 +38,9 @@ def setUp(self):
                         "verbosity": 0
                     }""")
                 ProjectParameters["solver_settings"].AddValue("linear_solver_settings", default_lin_solver_settings)
+            if not ProjectParameters["solver_settings"].Has("silent_warnings_block_builder_and_solver"):
+                silent_warnings_block_builder_and_solver = KratosMultiphysics.Parameters("""{"silent_warnings_block_builder_and_solver" : true}""")
```

Comment:
@RiccardoRossi do you agree with this?

Reply:
This is in #5905, better to comment there